models/users.js

```diff
@@ -38,6 +37,8 @@ exports.init_table = function (sequelize) {
             type: Sequelize.DATE,
             allowNull: true,
         },
+
+        // TODO: no need for this?
```

Comment:
Change this to number of late items attributed to this user, and then we can disable account if reaches a certain threshold.